dss/util/auth/auth0.py

```diff
@@ -41,27 +57,39 @@ def security_flow(self, **kwargs):
         executed_method(**kwargs)
 
     def _create(self, **kwargs):
-        """Auth checks for any 'create' API endpoint actions"""
+        """Auth checks for 'create' API actions"""
+        # Only check that a user is a member of a list of allowed organizations
         self.assert_required_parameters(kwargs, ['groups'])
         self._assert_authorized_group(kwargs['groups'])
         return
 
     def _read(self, **kwargs):
-        # Data is public by default
-        pass
-
-        # Eventually we will do a FLAC lookup first.
-        # If FLAC lookup doesn't raise exceptions,
-        # the requested read access will be granted.
-
-        # args/kwargs should include args/kwargs from
-        # both the decorator and the decorated function,
-        # so that's how we can get requested UUID
+        """Auth checks for 'read' API actions"""
+        # Data is public if there is no FLAC table entry.
+        self._assert_authorized_flac(**kwargs)
+        return
 
     def _update(self, **kwargs):
-        # Requires checking ownership of UUID
+        """Auth checks for 'update' API actions"""
+        try:
```

Comment:
Using this try/except structure to allow all admins feels a bit awkward. Maybe instead, we can check whether a user is an admin in the `_assert_authorized_flac()` method of the FLAC mixin class. Most actions should probably allow admins explicitly anyway.

Reply:
Fixed with #135 (adds a decorator to allow admins to bypass security checks)